README.md

```diff
@@ -30,6 +31,10 @@ This repository manages the documentation for the [Filecoin network](https://fil
 
 Want to help out? Pull requests (PRs) are always welcome! If you want to help out but aren't sure where to start, check out the [issues board](https://github.com/filecoin-project/filecoin-docs/issues).
 
+### Link checking
+
+Links are checked using [lychee-action](https://github.com/lycheeverse/lychee-action) as confirugred by [check-external-links.yml](.github/workflows/check-external-links.yml).  Working links are required before merging.  If you have a link that should be excluded from checking use [`.lycheeignore`](https://github.com/lycheeverse/lychee-action?tab=readme-ov-file#excluding-links-from-getting-checked) 
```

Comment:
I would add that raw links need backticks and not `<code>` blocks around them to work with the link checker.

Reply:
@lanzafame : i think `<code>` block works too.  I have updated the README.

Reply:
Ahh good to know